src/index.html

```diff
@@ -14,14 +14,15 @@
 	</style>
 	<div class="container">
 		<header>
-			<a class="mklive" href="live"><h2>We're live at milkenknights.com/live</h2></a>
+			<a class="mklive" href="live"><h2></h2>We're live at milkenknights.com/live</a>
```

Comment:
This is not what I was saying. The comment said:

> You seem to have moved the words "We're live at" inside of the <a> tag. You should love that text back out. That's what was causing your error in Atom.

The line should read `<h2>We're live at<a class="mklive" href="live">milkenknights.com/live</a></h2>` in full.